voice/voice-api/transfer-call-inline-ncco.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+source "../../config.sh"
+source "../../jwt.sh"
+
+curl -X POST https://api.nexmo.com/v1/calls/$UUID \
```

Comment:
```suggestion
curl -X PUT https://api.nexmo.com/v1/calls/$UUID \
```
This needs to be a PUT request.

Reply:
Yes. My bad. Fixed.